draft-irtf-panrg-path-properties.md

```diff
@@ -254,6 +254,7 @@ An IP router could be transparent to transport protocol headers such as TCP/UDP
 A firewall that only allows outgoing TCP connections by blocking all incoming TCP SYN packets regardless of their IP address is transparent to IP but not to TCP headers.
 Finally, a NAT that actively modifies IP and TCP/UDP headers based on their content is not transparent to either IP or TCP/UDP headers.
 Note that according to this definition, a node that modifies packets in accordance with the endpoints, such as a transparent HTTP proxy, as defined in {{RFC2616}}, and a node listening and reacting to implicit or explicit signals, see {{RFC8558}}, are not considered transparent.
+Transparency only applies to nodes and not to links, as a link cannot modify or perform any other actions on the packets by itself. For example, if the content of a packet is altered when forwarded over a Generic Routing Encapsulation (GRE) tunnel {{?RFC2784}}{{?RFC7676}}, this document considers as nodes the software instances that terminate the tunnel over which the actions are performed; thus, the transparency definition applies to these nodes.
```

Comment:
The text looks good to me. I would suggest to add a paragraph to not have such a big wall of text:
```suggestion

  Transparency only applies to nodes and not to links, as a link cannot modify or perform any other actions on the packets by itself. For example, if the content of a packet is altered when forwarded over a Generic Routing Encapsulation (GRE) tunnel {{?RFC2784}}{{?RFC7676}}, this document considers as nodes the software instances that terminate the tunnel over which the actions are performed; thus, the transparency definition applies to these nodes.
```